moira.py

```diff
@@ -10,19 +10,28 @@
 import re
 
 import _moira
-from _moira import (auth, host, motd, noop, proxy, MoiraException)
+from _moira import auth, host, motd, noop, proxy, MoiraException
 
 
-help_re = re.compile('([a-z0-9_, ]*) \(([a-z0-9_, ]*)\)(?: => ([a-z0-9_, ]*))?',
-                     re.I)
-et_re = re.compile(r'^\s*#\s*define\s+([A-Za-z0-9_]+)\s+.*?([0-9]+)')
+help_re = re.compile("([a-z0-9_, ]*) \(([a-z0-9_, ]*)\)(?: => ([a-z0-9_, ]*))?", re.I)
+et_re = re.compile(r"^\s*#\s*define\s+([A-Za-z0-9_]+)\s+.*?([0-9]+)")
 
 
 _arg_cache = {}
 _return_cache = {}
 _et_cache = {}
 
 
+def _to_bytes(s):
+    """
+    If given a string, converts it to bytes, otherwise returns it as is
+    """
+    if isinstance(s, str):
+        return s.encode()
+    else:
```

Comment:
Remove the else and unindent the `return s` since you early-return on line 30

Reply:
Done